README.md

```diff
@@ -12,13 +12,31 @@ CTFNote is a collaborative tool aiming to help CTF teams to organise their work.
 
 Before starting, make sure to fill in the information in the `.env` file.
 
-Then you can start it with `docker-compose`. The default
+### Pre-build images
+
+Building CTFNote requires at least 3 GB of RAM. If you want to host CTFNote
+on a server with less than 3 GB of RAM, you can use the pre-build images
+from the GitHub Container Registry.
+
+Make sure to [authenticate Docker to GHCR](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic).
```

Comment:
Is this really required ? Isn't this only for pushing and pulling from private repos ?
I could download your packages without providing a token.

Command: `docker image pull ghcr.io/jj-8/ctfnote/db`
Maybe I logged in a long time ago and don't remember it ?

Reply:
I remember that I had authorization issues in the beginning with GitHub when pulling the images. I also saw [this](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry) in the documentation:
> You need an access token to publish, install, and delete private, internal, and public packages.

But if it is actually not required, we can drop the requirement of course.

Reply:
I am quite confused. The document you quoted appears to say that a token is required, but I can download image without being authenticated.
Your tokens are stored in `~/.docker/config.json`. Can you try to download your image with no tokens ?
If you can confirm that pulling an image does not require a token, please remove this part from the documentation